src/prototype_creep_resources.js

```diff
@@ -280,6 +280,26 @@ Creep.prototype.pickupEnergy = function() {
     }
   }
 
+  // FIND_TOMBSTONES and get them empty first
+  const tombstones = this.pos.findInRange(FIND_TOMBSTONES, 1, {
+    filter: (t) => {
+      return t.creep.my;
```

Comment:
I think you can also withdraw from others, so you don't need to filter.

Reply:
good point